EarlGrey/Action/GREYActions.m

```diff
@@ -262,6 +266,31 @@ @implementation GREYActions
 #pragma mark - Private
 
 /**
+ *  Set the UITextField text value directly, bypassing the iOS keyboard.
```

Comment:
I'll work on updating this to be inlined into actionForClearText since that'll provide webview support.